### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by echee-wish**

### CODE REVIEW

File: cmd/root.go [hunk 1]
```diff
@@ -4,10 +4,12 @@ import (
 	"os"
 
 	"fmt"
+
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 	configcmd "github.com/wish/ctl/cmd/config"
-	"github.com/wish/ctl/cmd/kron"
+
+	//"github.com/wish/ctl/cmd/kron"
```
Comment (mlumibao-wish) [inline cmd/root.go]: Should get rid of this comment

Comment (mlumibao-wish) [reply cmd/root.go]: and update readme after too

**Comment by mlumibao-wish** [comment]
the comment is still there and make sure to run `go test` again after the gofmt

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,22 @@
+
+MIT License
+
+Copyright (c) 2019 ContextLogic Inc.
```
Comment (zelin-l) [inline LICENSE]: Change to 2020

**Comment by zelin-l** [comment]
Took a look at the code that was deleted - the deletion of the Kron web UI is good since it is not used (this is what I thought you were referring to before). Not sure about the deletion of exec/suspend/unsuspend because they would work in general for someone who wants to use ctl to manipulate the cronjobs. I'd advise to keep them and just rename package "kron" to just "cronjob"

File: README.md [hunk 3]
```diff
@@ -125,9 +125,6 @@ Ctl prints out the command before running.
 ## Config
 Cluster level configs are cached. To update this cached data, run `ctl config fetch`. Sometimes you may need to delete the ctl config folder. This is normally at `$XDG_CONFIG_DIR/ctl` or `~/.config/ctl`.
 
-## Kron
```
Comment (zelin-l) [inline README.md]: I scanned through the code again and I actually couldn't find base ctl commands that cover suspend/unsuspend for cron jobs. I think exec may be covered with the run command

File: README.md [hunk 4]
```diff
@@ -125,8 +125,8 @@ Ctl prints out the command before running.
...
-Kron features are outdated. Most cronjob features can be accessed through the base ctl commands. However, `ctl kron web` still remains useful. It creates a webserver to display all cronjobs. You may see `ctl help kron` for more details. Locally, this should be run in the ctl repo to access the static files.
+## Cron
+Most cronjob features can be accessed through the base ctl commands. The `ctl cron` command allows for diect manipulation of k8s cronjobs. You may see `ctl help crom` for more details.
```
Comment (zelin-l) [inline README.md]: typo for crom

**Pull request merged**